src/components/molecules/UserProfilePicture/UserProfilePicture.scss

```diff
@@ -1,83 +1,152 @@
 @import "scss/constants.scss";
 
 $fadedWhite: rgba(255, 255, 255, 0.8);
-$borderWidth: 5px;
 
-.profile-picture-container {
-  position: relative;
+// TODO: this was copied from Audience.scss (do we want to move the variable to constants?)
+$avatarSize: max(4vh, 3vw);
+
+// TODO: should we move these to using CSS --variables rather than these SCSS vars
+$reactionContainerOffset: calc(-1 * #{$avatarSize});
+$reactionSize: calc(#{$avatarSize / 1.5});
+$reactionOffset: calc(-1 * #{$reactionSize} / 2);
+$shoutBounceTranslateXOffset: calc(#{$avatarSize} / 2);
+
+.UserProfilePicture {
+  width: 100%;
+  height: 100%;
+
   background-position: center;
   background-size: cover;
 
-  .profile-icon {
-    width: 44px;
-    height: 44px;
-    background-size: 100%;
+  &--reaction-left {
+    .UserProfilePicture__reaction {
+      left: $reactionOffset;
+    }
+
+    .UserProfilePicture__shout {
+      right: 0;
+      transform-origin: right center;
+      animation: shout-bounce--left 5s ease;
+    }
+  }
+
+  &--reaction-right {
+    .UserProfilePicture__reaction {
+      right: $reactionOffset;
+    }
+
+    .UserProfilePicture__shout {
+      left: 0;
+      transform-origin: calc(#{$avatarSize} / 2) center;
+      animation: shout-bounce--right 5s ease;
+    }
+  }
+
+  &__avatar {
+    width: $avatarSize;
+    height: $avatarSize;
+
+    background-color: transparent;
+    // TODO: how can we use a default image here if none set? Can we use CSS --variables here? I think we could..
+    //background-image: url(${({ backgroundImage }) => backgroundImage ?? DEFAULT_PROFILE_IMAGE});
```

Comment:
[TODO] how can we use a default image here if none set? Can we use CSS --variables here? I think we could..

Reply:
See also https://github.com/sparkletown/sparkle/pull/1323/commits/db15bd47191b6c10594877933b651616c3375b67 + https://github.com/sparkletown/sparkle/pull/1323/commits/c3a2f885a4b5391c78e677009d55ba47238ab702 + https://github.com/sparkletown/sparkle/pull/1323/commits/f1469b499cbbb22ffcc4acb88790de65a3e1ddd8

Reply:
fixed in https://github.com/sparkletown/sparkle/pull/1323/commits/f0d70b78566230b1b6a33be0386e68aa45219450